### DISCUSSION TITLE
Overwrite default theme

### DISCUSSION THREAD

**Discussion by mammadataei** [unocss/unocss#1778] [Q&A] [answered]
Hi, How can I overwrite the default theme in `preset-mini` or `preset-wind` instead of deep merge? I don't need the default colors and want to overwrite them with my color palette. The same is true for `fontSize`, `fontWeight`, etc.

**Comment by chu121su12** [comment, marked as answer]
You can access and modify the merged theme in `extendTheme`
[Playground](https://uno.antfu.me/play/?html=DwEwlgbgBAxgNgQwM5ILwCIBGBzAtATwFM44B7Ad3QD4A5UgF1lLICdCRgB6cCKoA&config=JYWwDg9gTgLgBAbwFBzgEwKYDNgDsMDCEuOA5gDQpxhQYDOGMAqrhJQL5xZQQhwDkAV1YBjOnX5IkGAB6RY6bAENBAG3iYc%2BIiWCkAFMlQ16jOgC44AbSrHaDZq30BKSqgC6buLJgZcaABUACwwQDEt9GGc4AF4APkRbOAB5ACMAKwwRGAA6AGsMAE86SOccrGgAUSURIP082ISYKzz3WLhcNVVnAG4qdg5nJCA&options=N4IgzgLgTglgxhEAuaBXApgXyA)

**Reply by hrishikesh-k** [reply]
This doesn't seem to work for all utilities like width and height. The default ones are still generated. [Here's a modified version](https://uno.antfu.me/play/?html=DwEwlgbgBAxgNgQwM5ILwCIBGBzAtATwFM44B7AdynNwDMBXE9APgDlSAXWUsgJ0JGAB6cBCZA&config=JYWwDg9gTgLgBAbwFBzgEwKYDNgDsMDCEuOA5gDQpxhQYDOGMAqrhJQL5xZQQhwDkAV1YBjOnX5IkGAB6RY6bAENBAG3iYc%2BIiWCkAFMlQ16jOgC44AbSrHaDZq30BKSqgC6buLJgZcaABUACwwQDEt9GGc4AF4APkRbOAB5ACMAKwwRGAA6AGsMAE86SOccrGgAUSURIP082ISYKzz3WLhcNVVnAG4qdg5nJCA&css=PQKgBA6gTglgLgUzAYwK4Gc4HsC2YDCAyoWABYJQIA0YAhgHYAmYcUD6AZllDhWOqgAOg7nAB0YAGLcwCAB60cggDYIAXGBDAAUKDBi0mXGADe2sGC704AWgDuCGAHNScDQFYADJ4Dc5sAACtMLKAJ5gggCMLPK2ABR2pPBIcsoAlH4WAEa0yADWTlBYqEw2yFjK3Bpw5LxxAOTllVDoYpSMYgAs3vUZ2gC%2BmsBAA&options=N4IgzgLgTglgxhEAuaBXApgXyA) of your playground link reproducing the issue. Is this expected?
